src/components/pages/PointsDashboard/PointsDashboard.tsx

```diff
@@ -130,6 +131,12 @@ const ReferralCode = ({ code }: { code: string }) => {
 };
 
 const UserPointInfo = ({ chainId }: { chainId: number }) => {
+    const {
+        user: { loading: loadingUser, data: userData },
+        verifiedData,
+        verify,
+        loading,
```

Comment:
It's confusing since there are 2 `loading`.
It looks like `verifiedData`, `verify`, and `loading` can be grouped into one.
How about wrapping them by an object named `verification` or something like that?